src/components/SignInSteps/AccountStep.style.ts

```diff
@@ -36,7 +36,7 @@ export const SubTitle = styled(Text)`
 
 export const AccountsWrapper = styled.div`
   width: 100%;
-  max-height: 300px;
+  max-height: 200px;
```

Comment:
ok, so we have max-height and min-height with the same value. We could just replace them with just `height`, right?